pkgs/development/libraries/physics/geant4/datasets.nix

```diff
@@ -1,21 +1,21 @@
 { lib, stdenv, fetchurl, geant_version }:
 
 let
-  mkDataset = { name, version, sha256, envvar }:
+  mkDataset = { pname, version, sha256, envvar }:
```

Comment:
Is there an update script we need to adjust?

Reply:
i am unable to find one.